src/host/js/utils/enquire.js

```diff
@@ -0,0 +1,29 @@
+import enquire from 'enquire.js';
```

Comment:
the lib is 2.5kb. This whole thing will be combined with a new way of loading the load with an async script loader like what GA is doing so page load is not affected too much by our lib.